### DISCUSSION TITLE
Idea to provide Nuxt 3 for primevue template

### DISCUSSION THREAD

**Discussion by ace-featherlight** [primefaces/community#37] [PrimeVue Templates] [answered]
I have some idea to primetek team, on all PrimeVue template
- Nuxt 3 + TS
- Nuxt page / layout transition : this will unlock primevue template to have dropdown transition on template option
- [Nuxt Eslint](https://nuxt.com/modules/eslint) for linting and add prettier config too
- [Nuxt i18n](https://nuxt.com/modules/i18n) : all labeling is place in here, this will unlock all primevue template to have multilanguage feature 👍 
- [Nuxt Vitest](https://nuxt.com/modules/vitest) : To be honest I dont have example to test, so will be great if all template will have testing unit, so we can follow best practice from primeTek team
- [Nuxt Content](https://nuxt.com/modules/content) : the example of datatable, etc is using nuxt content, this will generate api to, so we can mimic endpoint fetching using nuxt fetch like to API
- [Nuxt Color Mode](https://nuxt.com/modules/color-mode ) : for manage light, dark, or system mode
- [Nuxt Icon](https://nuxt.com/modules/icon) : The icon is using nuxt-icon module, this is using [icones](https://icones.js.org) icon, don't worry I find [prime icon](https://icones.js.org/collection/prime) here, the benefit is you can add more tons of icon with this module 🔥 
- This is not related to nuxt, but It will great for template option, to give container options like : boxed or fluid, this is can doing easy, but is great if the template option add this selection

I think it's going to be a big effort, because you have so many good templates, and you also mange vite skeleton and nuxt
It's just an idea not necessarily, but will great if this included on primetek template roadmap for future, the template will more future proof, what do you think @cagataycivici ?

Thanks

**Comment by cagataycivici** [comment]
Interesting coincidence, here at 5am I'm updating the Nuxt quickstart sample to latest PrimeVue thinking we should roll out an official Nuxt module and consider using Nuxt more on templates.

We can definitely try this for a popular template like Apollo, btw Sakai is free and open source comes with [Vite](https://github.com/primefaces/sakai-vue) and [Nuxt](https://github.com/primefaces/sakai-nuxt) versions. So community contributions will be greatly appreciated. If the demand is high we can definitely apply it to other templates but a Nuxt version of Apollo is a very good candidate due to popularity.

**Reply by donPuerto** [reply]
Wow, thank you for this, it is a big step for the beginners.

**Comment by tugcekucukoglu** [comment, marked as answer]
Apollo-Vue 5.0.0 is released with Nuxt 3 support. https://apollo.primevue.org/documentation

You could reach from store now.
